frontend/src/constants.js

```diff
@@ -12,7 +12,7 @@ export const EXPANDED_WORKSHEET_WIDTH = '99%';
 export const DEFAULT_WORKSHEET_WIDTH = '65%';
```

Comment:
We should rename this to NARROW_WORKSHEET_WIDTH, as it's no longer the default.

Reply:
good catch. Renamed it.